src/ExpenseTrackerGrupo2/DataAccess/Layer/BaseRepository.cs

```diff
@@ -0,0 +1,47 @@
+using Dapper;
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.Data;
+
+namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Layer;
```

Comment:
Would you mind to wrap theses classes with a folder called `Concretes`? I suggest you due to repository pattern makes a reference of it.

Reply:
Done